_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+Author: Levi Blinder
+Title: Arts and Sciences Recipe Page
+---
+
```

Comment:
Would you please share the source for this recipe? Add link to the document here?